generators/client/templates/angular/src/main/webapp/app/shared/tracker/_tracker.service.ts

```diff
@@ -11,6 +10,9 @@ import { CSRFService } from '../auth/csrf.service';
 import { AuthServerProvider } from '../auth/auth-jwt.service';
 <%_ } _%>
 
+import SockJS = require('sockjs-client');
```

Comment:
I've tried that ones, but it didn't worked. Unfortunately sockjs-client exports only a constructor function, not an object with various functions on it. 
In plain javascript:
var sockjs = required("sockjs-client");
var s = new sockjs();
Instead of 
var s = new sockjs.SockJS();